PSReadLine/BasicEditing.cs

```diff
@@ -31,26 +31,20 @@ public static void SelfInsert(ConsoleKeyInfo? key = null, object arg = null)
             {
                 if (count <= 0)
                     return;
-                if (count > 1)
```

Comment:
Am I correct that these changes have nothing to do with predictive suggestions?

If so, can you move this change to a new PR?

Reply:
You are right. It's a small refactoring of the existing code. I will move it to a new PR.

Reply:
@lzybkr The changes in `SelfInsert` is the only refactoring code that is not directly related to the predictive suggestion feature.
Given it's small enough, is it OK to keep it in this PR?

Reply:
Well it's not really good engineering to include unrelated fixes with new features. For example, if one wanted to revert the feature, they'd also lose this fix.

Reply:
Understood. I will revert changes in this method and submit a separate PR.